src/main/java/de/tum/in/www1/artemis/service/dataexport/DataExportExerciseCreationService.java

```diff
@@ -114,13 +120,12 @@ public void createExercisesExport(Path workingDirectory, long userId) throws IOE
      * Creates an export for a given programming exercise. Includes submission information, the repository from the VCS and potential plagiarism cases.
      *
      * @param programmingExercise the programming exercise for which the export should be created
-     * @param courseDir           the directory that is used for the course the exercise belongs to
+     * @param exercisesDir        the directory that is used for to store the exercises of the course the exercise belongs to
```

Comment:
imo "the exercises of the course the exercise belongs to" is unnecessarily complex.
```suggestion
     * @param exercisesDir        the directory where this exercise should be written into
```

Reply:
simplified it